src/common/perf_counters.h

```diff
@@ -140,6 +140,11 @@ class PerfCounters
 	  0; // no change, report avg over all time
       return (cur.second - last.second) / (cur.first - last.first);
     }
+    T real_time() const {
+      if (cur.first == last.first)
+        return 0;
+      return (cur.second - last.second) / (cur.first - last.first);
+    }
```

Comment:
do you mean we just change the implementation and function name of avg()? for example, rename avg() to real_time(), and implement it as my new change?